packages/sanity/playwright-ct.config.ts

```diff
@@ -55,20 +57,20 @@ export default defineConfig({
     /* Port to use for Playwright component endpoint. */
     ctPort: 3100,
     /* Configure Playwright vite config */
-    /*
     ctViteConfig: {
       resolve: {
-        alias: {
-          '@sanity/util/content': path.join(
-            __dirname,
-            './packages/@sanity/util/src/content/index.ts'
-          ),
-        },
+        alias: Object.fromEntries(
+          Object.entries(aliases).map(([pkgName, pkgPath]) => {
+            return [pkgName, path.resolve(monorepoPath, path.join('packages', pkgPath))]
+          }),
+        ),
+        dedupe: ['@sanity/ui', 'styled-components'],
```

Comment:
this fixes [duplicate context warnings](https://www.sanity.io/help/missing-context-error) that are logged to playwright atm